### ISSUE TITLE
Missing FB preview

### ISSUE THREAD

**Issue by Beck-berry** [stadtnavi/digitransit-ui#230]
Herrenberg's Facebook preview looks like this:
![Screenshot_2020-06-09_09-59-34](https://user-images.githubusercontent.com/46535522/84121865-f2b39500-aa37-11ea-821b-d84b54624c78.png)
However we would like to display the image too as https://mih.mitfahren-bw.de/ and https://stadtnavi.de/ do. (Only host URL is different.)
```html
<meta data-react-helmet="true" property="og:image" content="https://herrenberg.stadtnavi.de/img/stadtnavi-social-media-card.png"/><meta data-react-helmet="true" property="og:image:width" content="600"/><meta data-react-helmet="true" property="og:image:height" content="300"/>
```

Using [Facebook Debugger Tool](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fherrenberg.stadtnavi.de%2F) I mostly got the error: `The 'og:image' property should be explicitly provided, even if a value can be inferred from other tags.` Although we have `og:image` defined. (Checked on other examples and the Debugger often says something is missing however it is given.)
Googled that and seems that there is no precise solution for it. Some said that 
* a space between the content and the closing tag
* no blank lines in the `head` part
* refreshing website's cache (but I don't think that this would the issue in our case since we lately updated the title and the description and those are the fresh ones)
* time ([SO issue comment](https://stackoverflow.com/a/42931857))
* adding `fb:app_id` meta property ([SO issue comment](https://stackoverflow.com/questions/10836135/when-do-i-need-a-fbapp-id-or-fbadmins#comment44959853_10836476))

can solve this problem. Since we are generating the meta tags within the `meta.js` file, we cannot explicitly handle whitespaces.

[SO issue](https://stackoverflow.com/questions/7869430/facebook-open-graph-error-inferred-property) with the most useful info.

We can scrape the info from time to time in the Debugger and see if it has any effect.

**Comment by leonardehrenfried** [comment]
![Screenshot from 2020-06-09 10-35-29](https://user-images.githubusercontent.com/151346/84125522-f39af580-aa3c-11ea-9f95-a633656f3c5b.png)

Weirdly, I get the result we want.

**Comment by leonardehrenfried** [comment]
And when I post, I see this:
![Screenshot from 2020-06-09 10-37-45](https://user-images.githubusercontent.com/151346/84125733-45dc1680-aa3d-11ea-91b0-df45756b81e6.png)

**Comment by Beck-berry** [comment]
Yes, for stadtnavi.de it works. But have a look on herrenberg.stadnavi.de.

**Comment by Beck-berry** [comment]
Also [`og:image:secure`](https://stackoverflow.com/questions/8855361/fb-opengraph-ogimage-not-pulling-images-possibly-https) might worth a try.

**Comment by Beck-berry** [comment]
Seems to be working now...
![Screenshot_2020-07-06_09-35-09](https://user-images.githubusercontent.com/46535522/86567670-0507ec80-bf6c-11ea-98ee-9926587fec99.png)

**Comment by leonardehrenfried** [comment]
![Screenshot from 2020-07-08 10-03-01](https://user-images.githubusercontent.com/151346/86893644-3aded980-c102-11ea-928a-58302072a51e.png)

For me it still does not work.

**Comment by leonardehrenfried** [comment]
![Screenshot from 2020-07-16 09-24-52](https://user-images.githubusercontent.com/151346/87639906-3cce1b80-c746-11ea-8d3a-e21789a45075.png)


It works now. :man_shrugging:
